interactive_engine/executor/ir/core/src/plan/logical.rs

```diff
@@ -1920,6 +1928,52 @@ mod test {
         );
     }
 
+    // e.g., g.V().hasLabel("person", "software").has("name", "John")
+    #[test]
+    fn scan_multi_labels_pred_to_idx_pred() {
```

Comment:
Lack a query case with result verification.

Reply:
Add a test case in test_min.py, which runs queries on vineyard that supports primary key.